src/main/java/io/github/externschool/planner/entity/profile/Profile.java

```diff
@@ -0,0 +1,161 @@
+package io.github.externschool.planner.entity.profile;
+
+import io.github.externschool.planner.entity.User;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "profile")
+@Inheritance(strategy = InheritanceType.JOINED)
+public class Profile {
```

Comment:
what about an abstract class with a minimal set of mapped Columns (profile_id, user_id)? 
inherited by:
* Guest: +phone Number only. email & password belong to the User, I'm not sure we should duplicate them with joins enabled; 
* Student, and Teacher with their specific data.

Reply:
can better be called a person without a profile